### PR TITLE
Decompile CEN EntityPrizeDrop, EntityEquipItemDrop

### PR DESCRIPTION
**Pull request by hohle**
These are the common implementation. They needed to be split for the jump tables to be positioned correctly.

### CODE REVIEW

File: config/splat.us.stcen.yaml [hunk 1]
```diff
@@ -39,7 +39,9 @@ segments:
       - [0xD42C, rodata]
       - [0xD4B0, .rodata, D600]
       - [0xD4D8, .rodata, 11280] # EntityRedDoor
-      - [0xD4F0, rodata]
+      - [0xD4F0, .rodata]
```
Comment (Xeeynamo) [inline config/splat.us.stcen.yaml:42]: I just cracked this one a couple of hours ago:

```c
static const char* g_goldCollectTexts[] = {
    _S("$1"),   _S("$25"),  _S("$50"),   _S("$100"),  _S("$250"),
    _S("$400"), _S("$700"), _S("$1000"), _S("$2000"), _S("$5000"),
};
```

**Review by Xeeynamo — APPROVED** [review]
I see some overlap between this PR and #1281 . I was normalising WRP and NZ0. It would be cool to rebase this PR on mine once merged and do the same with CEN.

Would you be happy to hold a tiny bit longer until #1281 gets merged? I am pretty much done with that one

**Comment by hohle** [comment]
> Would you be happy to hold a tiny bit longer until #1281 gets merged? I am pretty much done with that one

Absolutely!

File: config/splat.us.stcen.yaml (hunk 1 above)
Comment (hohle) [reply config/splat.us.stcen.yaml:45]: For my own knowledge, do these automatically get padded to 8 bytes (the smallest alignment of the longest string) or should this be declared as `char [][8]` to ensure alignment?

Comment (Xeeynamo) [reply config/splat.us.stcen.yaml:45]: I just checked. Yes, strings seem to be aligned by 8 bytes and padded automatically.

**Pull request merged**